importer/v8/importer.go

```diff
@@ -144,42 +144,49 @@ func (i *Importer) Import() error {
 	return nil
 }
 
-func (i *Importer) processDDL(scanner *bufio.Scanner) {
-	for scanner.Scan() {
-		line := scanner.Text()
+func (i *Importer) processDDL(scanner *bufio.Reader) {
+	for {
```

Comment:
There are a couple of issues here. (1) you could use `ReadString(byte('\n'))` instead of `ReadLine()`. That way you don't need to litter `string(line)` everywhere. Also, I think `ReadLine()` can return short reads, in that it might not return an entire line. 

The second thing is you're not checking for errors that are not nil, but also not `io.EOF`. Those errors would now need to be returned to the caller too.

How about we go with something like this?

```go
func (i *Importer) processDDL(scanner *bufio.Reader) error {
    for {
        line, err := scanner.ReadString(byte('\n'))
        if err != nil && err != io.EOF {
            return err
        }

	// If we find the DML token, we are done with DDL
        if strings.HasPrefix(line, "# DML") {
            return nil
        }

        if strings.HasPrefix(line, "#") {
            continue
        }

        // Skip blank lines
        if strings.TrimSpace(line) == ""  {
            continue
        }
        
        i.queryExecutor(string(line))

        // Check if this was the final token.
        if err == io.EOF { 
            return nil
        }
    }
    return nil
}
```

Reply:
Don't forget you'll need to add error handling to the call to `processDDL()` inside of `Import()`.